src/systems/user_interface/draw_user_interface.rs

```diff
@@ -0,0 +1,7 @@
+use bevy::ecs::event::EventWriter;
+
+use crate::events::user_interface_events::UserInterfaceEvent;
+
+pub fn draw_user_interface(mut draw_ui_event: EventWriter<UserInterfaceEvent>) {
+    draw_ui_event.send(UserInterfaceEvent {});
```

Comment:
@MatthewGriffin This event runs the updates that spawn the UI right?

I found something similar :smile:

Reply:
Yes the user interface event is what you trigger to create / refresh the ui